### DISCUSSION TITLE
Do structlog_gcp has any support for context vars?

### DISCUSSION THREAD

**Discussion by josiasrfreitas** [multani/structlog-gcp#34] [Q&A] [answered]
I want to add request_id's to my FastAPI application. It's actually like that:
 `structlog.configure(processors=build_processors())`
Is there any way to do that?

**Comment by multani** [comment]
@josiasdeveloper can you check https://github.com/multani/structlog-gcp/issues/33?

**Comment by multani** [comment, marked as answer]
@josiasdeveloper  I released https://github.com/multani/structlog-gcp/releases/tag/v0.3.0 which provides context variables support by default.

if you configure structlog with:

```python
processors = structlog_gcp.build_processors()
structlog.configure(processors=processors)
```

You should have context variables support automatically now. Hope it helps!
